stripecardscan/src/main/java/com/stripe/android/stripecardscan/cardscan/CardScanSheet.kt

```diff
@@ -0,0 +1,113 @@
+package com.stripe.android.stripecardscan.cardscan
+
+import android.content.Context
+import android.content.Intent
+import android.os.Parcelable
+import androidx.activity.ComponentActivity
+import androidx.activity.result.ActivityResultLauncher
+import androidx.activity.result.contract.ActivityResultContract
+import com.stripe.android.stripecardscan.cardscan.exception.UnknownScanException
+import com.stripe.android.stripecardscan.payment.card.ScannedCard
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+internal data class CardScanSheetParams(
+    val stripePublishableKey: String,
+//    val cardScanIntentId: String,
```

Comment:
nit: extra comments. 

@awush-stripe do we not lint in this part of the SDK for a reason?

Reply:
agreed, we can just remove these lines rather than leaving them commented.